utils/linux-install-luarocks.sh

```diff
@@ -16,11 +16,19 @@
 # limitations under the License.
 #
 
```

Comment:
are we not support luarocks 2.x?

Reply:
I think LuaRocks 3.x is better to use?

Reply:
agreed. So Apache APISIX not support luarocks 2.x, right?